content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -229,9 +231,8 @@ certain condition is true. The following taints are built in:
     as unusable. After a controller from the cloud-controller-manager initializes
     this node, the kubelet removes this taint.
 
-In version 1.13, the `TaintBasedEvictions` feature is promoted to beta and enabled by default, hence the taints are automatically
-added by the NodeController (or kubelet) and the normal logic for evicting pods from nodes
-based on the Ready NodeCondition is disabled.
+The taints are automatically added by the NodeController (or kubelet) and the normal
+logic for evicting pods from nodes based on the Ready NodeCondition is disabled.
```

Comment:
Is this also accurate
```suggestion
When the node controller or the kubelet observe problems on a node, they add taints to the
relevant Node object. If the fault condition returns to normal the kubelet or node
controller can remove the relevant taint(s). The kubelet reports problems that affect
its ability to run Pods whereas the node controller monitors whether the kubelet
itself is healthy.
```
?

I wouldn't mention “the normal logic for evicting pods from nodes based on the Ready NodeCondition is disabled” if there's no way to make use of that logic in v1.18. If there _is_ a way to make use of that formerly-normal logic, tell readers how to learn about that.

Reply:
We need to be careful here to properly distinguish when a taint has `NoSchedule` effect and when it has `NoExecute`. I have integrated the first two sentences. The last one expresses a general statement saying "a node is tainted based on a condition" which can have both effects interchangeable.